assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js

```diff
@@ -67,20 +68,24 @@
                         });
                         // Iterate over the checks to determine the validity of the step
                         vm.repoChecks.valid = Object.keys(vm.repoChecks.checks).every(function(k) {
+                            if (vm.repoChecks.checks[k].status === false) {
+                                repoErrors.push(vm.repoChecks.checks[k].label)
+                            }
                             return vm.repoChecks.checks[k].status === true
                         });
 
                         // if all the repos are ok, complete the step
                         if (vm.repoChecks.valid) {
                             upgradeStepsFactory.setCurrentStepCompleted();
+                        } else {
+                            // TODO(itxaka): This is a tmp thing, title and style should be clarified
+                            vm.error = { title: 'Some repositories are not ready', body: repoErrors.join(', ') };
                         }
                     },
                     // In case of failure
                     function (errorRepoChecksResponse) {
-                        // Expose the error list to repoChecks object
-                        // TODO(itxaka): Check the proper error response when the card is done
-                        // https://trello.com/c/chzg85j4/142-3-s49p7-error-reporting-on-crowbar-level
-                        vm.repoChecks.errors = errorRepoChecksResponse.data.errors;
+                        // TODO(itxaka): This is a tmp thing, title and style should be clarified
+                        vm.error = { title: 'Error while querying the API', body: errorRepoChecksResponse.data.errors }
```

Comment:
After recent changes leading to unify error formats around the application, expected format for error modal input is the "common format" used in other places.
See: https://github.com/crowbar/crowbar-ui/pull/117 and https://github.com/crowbar/crowbar-ui/pull/119